modules/installation-vsphere-infrastructure.adoc

```diff
@@ -41,6 +41,10 @@ You must ensure that the time on your ESXi hosts is synchronized before you inst
 |Optional: Networking (NSX-T)
 |vSphere 7.0 Update 2 or later; vSphere 8.0 Update 1 or later
 |At a minimum, vSphere 7.0 Update 2 or vSphere 8.0 Update 1 is required for {product-title}. For more information about the compatibility of NSX and {product-title}, see the Release Notes section of VMware's link:https://docs.vmware.com/en/VMware-NSX-Container-Plugin/index.html[NSX container plugin documentation].
+
+|CPU micro-architecture
+|x86-64-v2 or higher
```

Comment:
```suggestion
|x86-64-v2, ARMv8.0-A, Power9, Z14  or higher respective micro-architectures
```

or perhaps "for relevant micro-architectures" ? A cluster could be x86-64 only, x86-64 and ARM, etc. For whichever architectures are present the micro-architecture must be at least these versions.

Reply:
Hmm, just realized this is under vSphere, need to check to see if vSphere supports arches other than x86-64, checking...

Reply:
Confirmed at this time x86-64 is the only relevant arch for vSphere. I guess this could all be left as is, or in the right column you could remove references to Power9 and Z14, in this context.

Reply:
Thanks for checking on that. I've removed the references to power9 and Z14 from the right column.